proto/v1/payment.proto

```
@@ -0,0 +1,67 @@
+syntax = "proto3";
+
+package arco.v1;
```

Buf (1.54.0)

Files with package "arco.v1" must be within a directory "arco/v1" relative to root but were in directory "proto/v1".

(PACKAGE_DIRECTORY_MATCH)